Effisense/sensor-dashboard
# Feat/#43 All Sensors Map and Popover on marker

Closes: #43 

## Checklist


## Changelog
- New component AlllSensorsMap for viewing the location of all sensors in the db
- Query backend to get a list of all {lng, lat} from the database
- Add markers on the map to show location of the sensors (like we do in the useMap hook)
- Add popover on marker on map

## How to test
- Login and make sure you have a sensor connected to your user and that you are logged into the corresponding organization
- On the dashboard page there should be two maps, the bottom one is the new map that should have markers for each of your sensors
- Make sure that you know where your sensor is located to see the marker
- Click on the marker to see more information about the sensor
